src/DevBetterWeb.Core/Interfaces/IMemberSubscriptionPeriodCalculationsService.cs

```diff
@@ -10,5 +10,6 @@ public interface IMemberSubscriptionPeriodCalculationsService
     DateTime GetCurrentSubscriptionEndDate(Member member);
     DateTime GetGraduationDate(Member member);
     int GetPercentageProgressToAlumniStatus(Member member);
+    int GetPercentageProgressToAlumniStatus(int memberSubscribedDays);
```

Comment:
There's got to be an abstraction/data type you could use to return subscription details that doesn't involve so many primitive Get operations on this service. Unless there's a huge performance hit involved in any of these questions, just make a `SubscriptionDetails GetSubscriptionDetails(Member member)` call and then on that type add `CurrentSubscriptionEndDate` `GraduationDate` `PercentProgressToAlumniStatus` for instance.

Reply:
refactored out `GetPercentageProgressToAlumniStatus(int memberSubscribedDays);`
Better? 
This class _is_ supposed to be an abstraction to return subscription details.